fastly/backend_test.go

```diff
@@ -21,7 +21,6 @@ func TestClient_Backends(t *testing.T) {
 			ServiceVersion: tv.Number,
 			Name:           "test-backend",
 			Address:        "integ-test.go-fastly.com",
-			Port:           Uint(1234),
```

Comment:
ℹ️ I removed the setting of a port on creation so we can validate the API is setting a default of `80`.